custom_components/tuya_local/devices/topens_wifi_remote_control.yaml

```diff
@@ -0,0 +1,36 @@
+name: TOPENS Wi-Fi Remote control
+products:
+  - id: plgiarn7jndeu6js
+    name: TOPENS Wi-Fi Remote control
+primary_entity:
+  entity: button
+  name: A
+  dps:
+    - id: 101
+      type: boolean
+      name: button
+secondary_entities:
+  - entity: button
+    name: B
+    dps:
+      - id: 102
+        type: boolean
+        name: button
+  - entity: button
+    name: C
+    dps:
+      - id: 103
+        type: boolean
+        name: button
+  - entity: button
+    name: D
+    dps:
+      - id: 104
+        type: boolean
+        name: button
+  - entity: sensor
+    name: Limit Status
```

Comment:
What is this? Should it be `category: diagnostic`? Or even an attribute on another entity if it is unimportant.